web-client/src/views/FileHandlingHelpers/cloneFile.error.test.ts

```diff
@@ -29,6 +29,7 @@ describe('cloneFile', () => {
   it('should fail when attempting to clone something other than a file', async () => {
     let error;
     try {
+      // @ts-ignore
```

Comment:
Not even sure if this test is worth it since TypeScript should enforce the type?